chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -875,10 +875,12 @@ func (b *BitcoindNotifier) historicalSpendDetails(
 			}
 
 			txHash := tx.TxHash()
+			txCopy := tx.Copy()
```

Comment:
let's stick to the copy? So we should do `txHash := txCopy.TxHash()`

Reply:
Thanks! Updated